log.cpp

```diff
@@ -0,0 +1,79 @@
+#include "log.hpp"
+#include <iostream>
+#include <fstream>
+#include <ctime>
+using namespace std;

```

Comment:
It's best practice not to add `using namespace std`. If a function in the standard library has the same name as a function in our codebase, there will be conflicts.